Dshop/apps/products_catalogue/views.py

```diff
@@ -208,7 +165,7 @@ class CategoryDetailView(DetailView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        products_without_parent = Product.objects.filter(category=self.get_object(), parent_product=None, is_active=True)
+        products_without_parent = Product.objects.filter(category=self.get_object(), parent_product=None,
```

Comment:
For future reference: watch out for this formatting issues. You did not write this code, yet you will be visible in history, because of a formatting change. In such cases, you need to only commit the lines you changed. 
It's ok here, just a remark.

Reply:
I know, thanks! It sometimes happen, because of using ctrl+alt+L. I have to get rid of this habit 🙄